c_pal_ll/interfaces/reals/real_sync_renames.h

```diff
@@ -2,7 +2,10 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 #define wait_on_address        real_wait_on_address
+#define wait_on_address_64        real_wait_on_address_64
 #define wake_by_address_all    real_wake_by_address_all
+#define wake_by_address_all_64    real_wake_by_address_all_64
 #define wake_by_address_single real_wake_by_address_single
+#define wake_by_address_single_64 real_wake_by_address_single_64
 
```

Comment:
Consider having the right side indented at the same column

Reply:
Done